frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/AddPolicyModal.tsx

```diff
@@ -91,23 +135,43 @@ const AddPolicyModal = ({
     );
   });
 
+  const renderNoResults = (): JSX.Element => {
+    return (
+      <>
+        There are no results that match your filters.{" "}
+        <CustomLink
+          text="Everyone can contribute"
+          url={CONTRIBUTE_TO_POLICIES}
```

Comment:
If this is the only place we use this URL, maybe makes sense to just put it right here without the variable?

Reply:
My hopes is to eventually make a file that organizes all our external URLs so I've just been adding them to the top of recent PRs as constants.